.editorconfig

```diff
@@ -0,0 +1,31 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
```

Comment:
add a `charset = utf-8` here ;P

Reply:
It is already present on line 12 .

Reply:
`[*]` means the file present in the present depth, so you need to add this.